src/WheelsAndGoods.Core/Models/Entities/User.cs

```diff
@@ -22,6 +22,8 @@ private User()
 	public string Firstname { get; private set; }
 	public string Lastname { get; private set; }
 	public string Phone { get; private set; }
+	public string AvatarUrl { get; set; }
```

Comment:
Good job. Also add rules for this fields in UserConfiguration (DataAccess) please.